R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
It's not required to be specified but when it is missing it is inferred from the function name, and it's used by roxygen2.
Do you have the generated HTML? it might help to isolate what's going on.